### PR TITLE
Update buienradar.markdown

### PR DESCRIPTION
**Pull request by Mettchen**
Changed wording of camera entity sentence. Added information on how to enable the camera entity.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/buienradar.markdown [hunk 1]
```diff
@@ -37,7 +37,7 @@ The `buienradar` camera platform uses [buienradar.nl](https://buienradar.nl/) as
 Internally, this integration uses the radar map image as [documented](https://www.buienradar.nl/overbuienradar/gratis-weerdata) on buienradar.nl.
 The downloaded image is cached to prevent Home Assistant from making a new request to buienradar.nl multiple times a minute when Home Assistant checks for new stills from the camera.
 
-The camera entity is added disabled by default and should first be enabled before it starts reading the camera images.
+The camera entity is disabled by default and should be enabled before it reads the camera images. The camera entity is enabled by going to Settings > Devices and Services > Entities. Search for "camera.buienradar". Click on the entity and, after that, the cogwheel. Then ensure that the slider under "Enabled" is on.
```
Comment (frenck) [inline source/_integrations/buienradar.markdown]: This applies to many many entities (of Buienradar and many others), not just the camera entity.

This is a more generic instruction of Home Assistant, to me it feels odd to add it just here?

../Frenck

Comment (Mettchen) [reply source/_integrations/buienradar.markdown]: That is an excellent point. I agree it's a more generic HA feature.
I tried looking for documentation through official channels, which I could then link to. Do you have a good idea of where to search for it in the official docs?

And you are right, all of them are disabled, except for weather.buienradar.

For ease of usage for newer users, in my opinion, it should be officially documented somewhere, which can then link to... Somewhat like https://github.com/home-assistant/home-assistant.io/blob/current/source/_includes/integrations/option_flow.md is used in some integrations.

Comment (c0ffeeca7) [inline source/_integrations/buienradar.markdown]:
```suggestion
The camera entity is disabled by default and should be [enabled](/common-tasks/general/#enabling-entities) before it reads the camera images. 
```

Comment (c0ffeeca7) [reply source/_integrations/buienradar.markdown]: Thank you for the suggestion. There is now a small procedure in the _Common tasks_ section of the documentation, describing how to enable an entity.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Mettchen :+1:

**Pull request merged**